pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.builds

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If this is about tools why name it after the meta-package as opposed to naming it like the toolchain package? I worry adding the .CoreCLR to the end of the meta-package will cause some confusion.